### ISSUE TITLE
We need webpack

### ISSUE THREAD

**Issue by kasvith** [sanjayaharshana/AnimTrap#20]
Gulp is very bad in this situation.
Webpack is more powerful and can do awesome job with it's dependency tree strategy. Lets use it

**Comment by sanjayaharshana** [comment]
Yes, it is so necessary.We'll make it to Dev branch.Then we'll switch it to the main branch at the right time
